### PR TITLE
Personal textsubs

### PR DESCRIPTION
**Pull request by jayRyan24**
Adding in a small change to let people use custom textsubs from individual chars. I thought of making it file-based, but this seems like a straight forward way each char can have separate textsubs.

### CODE REVIEW

**Comment by jayRyan24** [comment]
```
look moz
You are Jay, a Human.
You have a square face, tilted jade eyes and a straight nose.  You have burnt to a fiery crisp hair, with copper skin and a stout build.
You are tall for a Human.
```

```
private_textsubs:
  Mozof: Jay
  cropped auburn hair: burnt to a fiery crisp hair
```

File: textsubs.lic [hunk 1]
```diff
@@ -47,6 +47,12 @@ if get_settings.integrate_shit_list_with_textsubs && File.file?("#{checkname}_sh
   end
 end
 
+unless get_settings.private_textsubs.empty?
```
Comment (rpherbig) [inline textsubs.lic:50]: Not necessary for this PR, but it would be nice to cache `get_settings` instead of calling it 4+ times in this script.

**Comment by rpherbig** [comment]
✔️ Looks good - ready for me to merge it?

File: textsubs.lic (hunk 1 above)
Comment (jayRyan24) [reply textsubs.lic:50]: I have thought about that, and I did at first until I realize those were all outside of the textsubs class. How do you want me to handle it?

Comment (rpherbig) [reply textsubs.lic:50]: It can still be stored off into a var. But that's a problem for another day.

Comment (jayRyan24) [reply textsubs.lic]: OK, I pushed it!

**Comment by jayRyan24** [comment]
Ready

File: textsubs.lic (hunk 1 above)
Comment (jayRyan24) [reply textsubs.lic]: No clue why I was brain farting on the variable simply outside of the class, I'm doing too many things at once.

**Comment by jayRyan24** [comment]
> ✔️ Looks good - ready for me to merge it?

@rpherbig Ready!

**Comment by jayRyan24** [comment]
This can be merged @rpherbig, thanks!

**Pull request merged**